File: packages/expo/ios/ClerkOrganizationSwitcherViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(ClerkOrganizationSwitcherViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(hidePersonal, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(onOrganizationEvent, RCTBubblingEventBlock)

@end

File: packages/expo/ios/ClerkOrganizationSwitcherViewManager.swift
import React

@objc(ClerkOrganizationSwitcherViewManager)
class ClerkOrganizationSwitcherViewManager: RCTViewManager {

  override static func requiresMainQueueSetup() -> Bool {
    return true
  }

  override func view() -> UIView! {
    return ClerkOrganizationSwitcherNativeView()
  }
}
